src/assets/styles/theme.scss

```diff
@@ -19,12 +19,14 @@
 @import './global/typography';
 @import './global/rte';
```

Comment:
Do you think RTE should be a component also? If I understood correctly, we use "rte" only for the fields that have this enabled?